qemu/tests/cfg/ovmf_check_efi.cfg

```diff
@@ -2,7 +2,7 @@
     only q35
     only ovmf
     type = ovmf_check_efi
-    check_cmd = "gdisk -l /dev/[vs]da"
+    check_cmd = "parted -l /dev/[vs]da"
```

Comment:
hi @XueqiangWei , parted -l will list partition layout on all block devices, of course, in your case, only one device is configured,  can we remove  /dev/[vs]da?

Reply:
Updated, thanks Zhenchao.